tools/marine_omics/sanntis.xml

```diff
@@ -11,21 +11,49 @@
         <requirement type="package" version="@TOOL_VERSION@">sanntis</requirement>
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
-        sanntis --ip-file '$input_interpro' --outfile 'output_sanntis.gff' '$input_genbank'
+    #if $selection.which_sanntis == 'sanntis': 
+        sanntis --ip-file '$selection.input_interpro' --outfile 'output_sanntis.gff' '$selection.input_genbank'
+    #else:
+        sanntis_build_gb  -n '$selection.input_nuc' -a '$selection.input_prot' -o 'output_sanntis_gb.gb'
+    #end if 
     ]]></command>
     <inputs>
-        <param name="input_interpro" type="data" format="tabular" label="Input the TSV file from InterProScan" help="Before using this tool you need to retrieve the right data by using the InterProScan tool"/>
-        <param name="input_genbank" type="data" format="genbank" label="Input a Genbank .gb file" help="It needs to have the right structure and fit the protein fasta file used in InterProScan"/>
+        <conditional name="selection">
+            <param name="which_sanntis" type="select" label="Do you want to build a genbank or to use sanntis ?" help="If you decide to build a genbank you can then use this genbank intot sanntis.">
```

Comment:
This tool is Sanntis, correct? It might be confusing if you say ... do you wan to build GB or Sanntis...

Reply:
Is it better now ?